script.js

```diff
@@ -1,12 +1,142 @@
 //You can edit ALL of the code here
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
+  let allShow = getAllShows();
+  //Sorts shows in alphabetical order
+  allShow.sort(function (a, b) {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1;
+    }
+    return 0;
+  });
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
+    const homebtn = document.createElement("button");
+    homebtn.textContent = "Home";
+    homebtn.className = "home";
+    const search = document.querySelector("#search");
+    const searchInput = document.createElement("input");
+    const episodeNum = document.createElement("p");
+    search.appendChild(homebtn);
+    //displayPage(episodeList);
+    let selectShow = document.createElement("select");
+    selectShow.className = "selectshow";
+    search.appendChild(selectShow);
+    let selectEpisode = document.createElement("select");
+    selectEpisode.className = "selectepisode";
+    search.appendChild(selectEpisode);
+    searchInput.type = "text";
+    searchInput.placeholder = "Search";
+    search.appendChild(searchInput);
+    search.appendChild(episodeNum);
```

Comment:
Sometime, you may like to consider using append method instead of appendChild method because append( ) can append several nodes and strings. Really nice clean codes.  :)